index.html

```diff
@@ -52,6 +54,8 @@
       </div>
     </nav>
 
+    <!-- Adding a searchbar, because they're cool! -->
```

Comment:
Using bootstrap to create a search bar would make it even more better!

Reply:
Yes! This would ensure that out site continues to be responsive and we can easily adjust the search bar for smaller devices.